web/src/pages/Setting/Setting.tsx

```diff
@@ -75,6 +75,8 @@ const Setting: React.FC = () => {
               }
               history.push(path);
             }
+          } else {
+          history.push('/');
```

Comment:
you could run `yarn prettier --write .` under the `web` directory to format codes.

Reply:
you can also click https://github.com/apache/apisix-dashboard/pull/1284/checks?check_run_id=1696195316 to find out what lint error it is.

Reply:
![yarn-command](https://user-images.githubusercontent.com/48581379/104594063-a21fac80-5696-11eb-893c-5cedbae902c8.png)
@juzhiyuan, I could not able to execute the command inside the `web` directory

Reply:
Try running yarn install then have a try? It seems that the prettier Module is missing :)

Reply:
Sure, I'll try it and let you know